src/kata7.getEmployerRole.js

```diff
@@ -1,8 +1,9 @@
-
+// review
 
 
 
 const getEmployerRole = (name, employees) => {
+  // review
```

Comment:
You're indented 2 spaces too much inside this function. Also, you could make your life much easier (and your code much more succinct) by using [Array.prototype.find()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) here.